main.c

```diff
@@ -319,8 +319,11 @@ void drawShellInfo(char *path) {
 
 	vita2d_texture *battery_bar_image = battery_bar_green_image;
 
-	if (scePowerIsLowBattery())
+	if (scePowerIsLowBattery()){
 		battery_bar_image = battery_bar_red_image;
+	} else if (scePowerIsBatteryCharging()) {
+		battery_bar_image = battery_bar_charge_image;
```

Comment:
i think, this condition should be swap. if we charge device, low battery info is no more important.
if this info is important, we should split red-battery and red-charge-battery